bHY6yh-d11728/cod-yyyiou_bHY6yh-d11728.xml

```diff
@@ -0,0 +1,457 @@
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title>Rubrica 3</title>
+        <author ref="#per-c7y65a">Geremia da Montagnone</author>
+        <respStmt>
+          <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+          <resp>Transcription Editor</resp>
+          <resp>TEI Encoder</resp>
+        </respStmt>
+        <respStmt>
+          <name ref="#alasdairwsliter">Alasdair W. Sliter</name>
+          <resp>Transcription Editor</resp>
+          <resp>TEI Encoder</resp>
+        </respStmt>
+      </titleStmt>
+      <editionStmt>
+        <edition n="0.0.0-dev">
+          <title>Rubrica 3</title>
+          <date when="2023-06-30">June 30, 2023</date>
+        </edition>
+      </editionStmt>
+      <publicationStmt>
+        <authority>SCTA</authority>
+        <availability status="free">
+          <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+          </p>
+        </availability>
+      </publicationStmt>
+      <sourceDesc>
+        <listWit>
+          <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+        </listWit>
+      </sourceDesc>
+    </fileDesc>
+    <encodingDesc>
+      <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2023-06-30" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#C" n="16-v"/>
+        <cb ed="#C" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-d11728"><!-- p1l2-r3 -->
+        <head xml:id="bHY6yh-d11728-Hd1e108">Rubrica 3</head>
+        <head xml:id="bHY6yh-d11728-Hd1e111" type="question-title">De virtutibus et vitiis</head>
+        <p xml:id="bHY6yh-d11728-d1e114">
+          De virtutibus
+          <lb ed="#C"/>et uitiis
+        </p>
+        <!--the next paragraph starts on the same line as the previous paragraph-->
+        <p xml:id="bHY6yh-d11728-d1e119">
+          <ref xml:id="bHY6yh-d11728-Rd1e116">Plato in menone.</ref>
+          <quote xml:id="bHY6yh-d11728-Qd1e123">virtus est potentia 
```

Comment:
is there a new line here before the quote? If not then, there shouldn't be a hard return. If there is, then we are missing an `<lb/>`
```suggestion
          <ref xml:id="bHY6yh-d11728-Rd1e116">Plato in menone.</ref>
          <lb ed="#L"/><quote xml:id="bHY6yh-d11728-Qd1e123">virtus est potentia 
```